src/lib/state.ts

```diff
@@ -241,19 +267,19 @@ export const dumpNode = (
     status: node.status,
     ready: node.ready,
     isListening: node.isListening,
-    isFrequentListening: node.isFrequentListening,
+    isFrequentListening: Boolean(node.isFrequentListening),
     isRouting: node.isRouting,
-    maxBaudRate: node.maxBaudRate,
+    maxBaudRate: node.maxDataRate,
```

Comment:
Don't add this here since we're going to remove it in schema 3. Only here add the base that's the same across all versions.